examples/darwin-framework-tool/commands/clusters/ReportCommandBridge.h

```diff
@@ -241,3 +241,68 @@ class SubscribeEvent : public ModelCommand {
     uint16_t mMinInterval;
     uint16_t mMaxInterval;
 };
+
+class ReadEvent : public ModelCommand {
+public:
+          ReadEvent()
+        : ModelCommand("read-event-by-id")
+    {
+        AddArgument("cluster-id", 0, UINT32_MAX, &mClusterId);
+        AddArgument("event-id", 0, UINT32_MAX, &mEventId);
+        AddArgument("fabric-filtered", 0, 1, &mFabricFiltered);
```

Comment:
This part looks like it's duplicating chip-tool, but event reads ignore this fabric-filtered boolean, no?  So we should not have it here, for any of these functions.

Reply:
In my understanding, the fabric-filter option is needed for all matter commands including read event. We tested that the option is working in read event scenario. If it is misunderstanding of the option, please let me know. I will remove the option in the functions. Our test sequence is as follows:

-Pre-condition
     1. Paring is completed between lock-app and darwin-framework-tool.
     2. Generate door-lock-alarm event with "echo '{"Cmd": "SendDoorLockAlarm", "Params": { "EndpointId": 1, "AlarmCode": 0 } }' > /tmp/chip_lock_app_fifo-<PID>   (PID of lock app)"

-Test sequence
    1. Read door-lock-alarm event with fabric-filter option 0 or 1: "doorlock read-event-by-id 0 123456 1 --fabric-filtered 0 or 1"
   2. Check lock-app log whether isFabricFiltered value in ReadRequestMessage is set as expected: 0 -> false and 1 -> true.

Reply:
The "fabric filtered" option only affects the behavior of reading:

1. Attributes
2. That are of type list.
3. Where the entries in the list are structs
4. Where the structs are fabric-scoped.

It has no impact on reading events at all.  That is, you can send the boolean in the request message, but it will not change what data you get back as a result...

Reply:
Maybe it seems my misunderstanding. I have thought that the option affects the other fabric because chip-tool read-event also has that option. I will double check the meaning of the option and actual result in multi-fabric environments. I will update soon after that.

Reply:
@bzbarsky-apple I removed fabric-filtered option in read-event-by-id of darwin-framework-tool.